src/js/translations/en-US.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
When this is merged I'm going to add a PR suggestion to`CONTRIBUTING` where will explain how to add i18n IDs.